crates/ark/src/shell.rs

```diff
@@ -229,24 +203,6 @@ impl ShellHandler for Shell {
             ExecuteResponse::ReplyException(err) => Err(err),
         };
 
-        let mut kernel = self.kernel.lock().unwrap();
```

Comment:
I really really like that we now just:
- send request
- get response
- return response

With no extra funny business